app/tests/unit/lib/theme/widgets/ContextualHelp.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from "@testing-library/react";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import ContextualHelp from "lib/theme/widgets/ContextualHelp";
+
+describe("ContextualHelp component", () => {
+  const textMock = "Tooltip Text";
```

Comment:
I'd suggest adding html to the text mock and checking that things look right after it gets parsed. There could also be another test with malformed html to see if the parser handles that well

Reply:
That's a good point I've added an extra test for HTML text, Thanks 👍 I think the second test is not needed since we don't want to test the package itself.